packages/utilities/psammead-test-helpers/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 4.0.0 | [PR#3180](https://github.com/bbc/psammead/pull/3180) Bump styled components |
```

Comment:
This change bumps `jest-styled-components` and _adds_ `styled-components` as a dependency. This line should probably reflect that.